rollup.config.js

```diff
@@ -38,7 +38,7 @@ export default [
   {
     ...cfg,
     output: {
-      file: './build/holoplay-webxr.js',
+      file: './build/LookingGlass-webxr.js',
```

Comment:
we may want this to be `looking-glass-webxr.js` instead, the current `LookingGlass-webxr.js` does work and doesn't cause any technical issues though.

Reply:
Agreed, I prefer looking-glass-webxr.js. Can we rename it that? If we're renaming, might as well go with the cleanest approach. Lmk if there's some challenge or disadvantage in doing so.